Makefile

```diff
@@ -87,10 +87,45 @@ macos:
 android:
 	@$(call print, "Building Android library ($(ANDROID_BUILD)).")
 	mkdir -p $(ANDROID_BUILD_DIR)
-	GOOS=js $(GOMOBILE_BIN) bind -target=android -tags="mobile $(DEV_TAGS) $(RPC_TAGS)" -androidapi 21 $(LDFLAGS_MOBILE) -v -o $(ANDROID_BUILD) $(MOBILE_PKG)
+	GO386=softfloat GOOS=js $(GOMOBILE_BIN) bind -target=android -tags="mobile $(DEV_TAGS) $(RPC_TAGS)" -androidapi 21 $(LDFLAGS_MOBILE) -v -o $(ANDROID_BUILD) $(MOBILE_PKG)
 
 mobile: ios android
 
+repro-wasm:
+	#Build the repro-wasm image
+	docker build -f Dockerfile-wasm -t repro-wasm-image --no-cache .
+
+	#Run the repro-wasm-image in a new container called repro-wasm
+	docker run --name repro-wasm  repro-wasm-image
+
+	#Copy the compiled WASM file to the host machine
+	mkdir -p reproducible-builds
+	docker cp repro-wasm:/app/cmd/wasm-client/wasm-client.wasm ./reproducible-builds/
+	
+	#Remove the repro-wasm container
+	docker rm repro-wasm
+
+	#Remove the repro-wasm-image
+	docker image rm repro-wasm-image
+
+repro-android:
+	#Build the repro-android image
+	docker build -f Dockerfile-android -t repro-android-image --no-cache .
+
+	#Run the repro-android-image in a new container called repro-android
+	docker run --name repro-android repro-android-image
+
+	#Copy the built Android library files to the host machine
+	mkdir -p reproducible-builds/android
+	docker cp repro-android:/app/build/android/lnc-mobile.aar ./reproducible-builds/android
```

Comment:
This needs to be `/gomobile/src/github.com/lightninglabs/lightning-node-connect/build/android/lnc-mobile.aar` (and `.../lnc-mobile-sources.jar` respectively) since within the container the path is set to `$(ANDROID_BUILD_DIR)` which is `${GOPATH}/src/$(PKG)/build`.